source/actors/player/Item.hx

```diff
@@ -0,0 +1,80 @@
+package actors.player;
+
+import flixel.util.FlxTimer;
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+class Item extends FlxSprite 
+{
+    public var sWidth(default, never):Int = 50;
+    public var sHeight(default, never):Int = 10;
+
+    // 0 = right, 1 = down, 2 = left, 3 = up
+    private var lastUsed:Int = 0;
+
+    private var time:FlxTimer;
+    private var delay:Float = 0.2;
+
+    public function new() {
+        super();
+    }
+    override public function update(elapsed:Float):Void
+        {
+            if(FlxG.keys.pressed.RIGHT)
+                {
+                    lastUsed = 0;
+                }
+            else if(FlxG.keys.pressed.DOWN)
+                {
+                    lastUsed = 1;
+                }
+            else if(FlxG.keys.pressed.LEFT)
+                {
+                    lastUsed = 2;
+                }
+            else if(FlxG.keys.pressed.UP)
+                {
+                    lastUsed = 3;
+                }
```

Comment:
the logic here can get sketchy if the user presses multiple keys at the same time (if-else if-else block is all exclusive, so there's some built in priority going on that doesn't necessarily give us the result we want).

Instead of making the item responsible for keeping track of this "facing" information, I recommend using the hero's `facing` property (defined in the FlxSprite class: https://api.haxeflixel.com/flixel/FlxSprite.html#facing). That variable isn't updated automatically by anything in the game right now, but you can add logic to the movement direction-selecting function to make it set `this.facing = FlxObject.LEFT` or other directions as needed.